src/components/CounterInput/helpers/floatify.js

```diff
@@ -0,0 +1,5 @@
+const floatify = number => {
```

Comment:
think about another name more meaningful that start with `get` like getNormalizedValue